ui/components/multichain/pages/send/send.test.js

```diff
@@ -36,6 +36,19 @@ describe('SendPage', () => {
   describe('render', () => {
     it('renders correctly', () => {
       const { container, getByTestId } = render();
+      const expectedPlaceholder =
```

Comment:
Should we just unit test against main? Instead of this? I don't think anyone is going to be running Jest with the environment variable

Reply:
Not sure I follow, how would we just unit test against main?